src/components/landing_page/UserAuthNav.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import './UserAuthNav.scss';
+
+function UserAuthNav() {
+  return (
+    <div className="UserAuthNav">
+      <Link
+        to="/login"
+        className="UserAuthNav__item"
+        activeClassName="UserAuthNav__item--active"
+      >
+        Login
+      </Link>
+      <Link
+        to="/register"
+        className="UserAuthNav__item"
+        activeClassName="UserAuthNav__item--active"
```

Comment:
You are using `activeClassName` but keep in mind that only `NavLink` components have this property.
Just change `Link` to `NavLink` from `react-router-dom`.